src/app/modules/submission/email-submission/email-submission.util.ts

```diff
@@ -266,6 +267,7 @@ export const mapRouteError: MapRouteError = (error) => {
       }
     case DatabaseError:
     case SubmissionHashError:
+    case SendAdminEmailError:
```

Comment:
Should this be a 5xx? What if it's AWS screwing up? Our previous error was 400 right? Maybe we can follow the error codes SES returns: https://docs.aws.amazon.com/ses/latest/DeveloperGuide/using-ses-api-error-codes.html

Reply:
yeah, agree